config/routes.rb

```diff
@@ -27,6 +27,10 @@
   post "/update", to: "salt#update"
   post "/accept-minion", to: "salt#accept_minion"
 
+  resource :orchestration, only: [] do
+    match "/retry-bootstrap", action: :retry_bootstrap, via: [:put, :patch]
```

Comment:
do you think it makes sense also to expose a `retry-upgrade` endpoint?

Reply:
I think we should do this in the future, yeah... so we can allow to restart upgrade orchestrations from the UI.